src/ace/pdf_extraction.py

```
@@ -1,3 +1,32 @@
+    :returns: A string containing all the text extracted from the PDF
+    """
+    # Extracting the UploadFile object's content into byte format
+    # as UploadedFile is a temporary object and not an actual fle
+    pdf_byteio = streamlit_upload_object.getvalue()
+
+    # Check to intercept the edge cases of corrupted file or streamlit file mishandling
+    if not pdf_byteio:
+        raise ValueError(
+            "The uploaded PDF file is empty."
+            "This could be due to file being corrupted or upload interruption, please try uploading."
+        )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)